Documentation/cmdref/cilium-agent.md

```diff
@@ -42,6 +42,7 @@ cilium-agent
       --ipv6-service-range string         Kubernetes IPv6 services CIDR if not inside cluster prefix (default "auto")
       --k8s-api-server string             Kubernetes api address server (for https use --k8s-kubeconfig-path instead)
       --k8s-kubeconfig-path string        Absolute path of the kubernetes kubeconfig file
+      --k8s-use-node-annotations          Annotate kubernetes nodes to build routable mesh
```

Comment:
Right now they always match so it does not matter. The idea is to allow simple operation without a kvstore but it may be better to just do a CRD backend and never use annotations anymore.